### ISSUE TITLE
Condition the yes no for interactivity

### ISSUE THREAD

**Issue by ColinFay** [ThinkR-open/golem#100]
When in bash 

``` bash
MacBook-Air-de-colin:testgolem colin$ RScript -e "golem::create_golem('plop')"
── Created ─────────────────────────────────────────────────────────────────────
MacBook-Air-de-colin:testgolem colin$ RScript -e "golem::create_golem('plop')"
The path plop already exists, override?Error in menu(qs[rand]) : menu() cannot be used non-interactively
Calls: <Anonymous> -> <Anonymous> -> menu
Exécution arrêtée

```

**Comment by ColinFay** [comment]
This also goes for all the add_ functions

**Comment by ALanguillaume** [comment]
This was solved in #777
